controller/wingsail/wingsail_ctrl_node.py

```diff
@@ -44,6 +46,8 @@ def __init_private_attributes(self):
         during the initialization process.
         """
         self.__trim_tab_angle = 0.0
+        self.filtered_wind_sensor = None
+        self.gps = None
```

Comment:
Change these variables to have two underscores before them. This will effectively make these class members private. Without the double underscores, they are publically accessible, which is not what we want.

For example, `self.__gps` instead of `self.gps`

Reply:
changed them to __filtered_wind_sensor and __gps